nilearn/regions/signal_extraction.py

```diff
@@ -103,10 +116,11 @@ def img_to_signals_labels(imgs, labels_img, mask_img=None,
     data = _safe_get_data(imgs)
     signals = np.ndarray((data.shape[-1], len(labels)), order=order,
                          dtype=data.dtype)
+
     for n, img in enumerate(np.rollaxis(data, -1)):
-        signals[n] = np.asarray(ndimage.measurements.mean(img,
-                                                          labels=labels_data,
-                                                          index=labels))
+        signals[n] = np.asarray(measurement_func(img,
```

Comment:
`reduction_function` or `aggregation_function` instead of `measurement_func` since we are not directly measuring.
Or `reduction_strategy` if you decide to choose `strategy` s the parameter name.

Reply:
Here I went for reduction_function, as we are now dealing with a callable and not a str.